recipe/build_base.bat

```diff
@@ -2,8 +2,10 @@ setlocal EnableDelayedExpansion
 echo on
 
 :: brand Python with conda-forge startup message
-%SYS_PYTHON% %RECIPE_DIR%\brand_python.py
-if errorlevel 1 exit 1
+if "%CONDA_FORGE%"=="yes" (
+  %SYS_PYTHON% %RECIPE_DIR%\brand_python.py
+  if errorlevel 1 exit 1
+)
```

Comment:
Can this script just be replaced in the Anaconda case? IOW is the condition needed?

Reply:
Being baked into the C code, the anaconda method is likely faster and I believe our performance lead in startup tests is largely down to this. Still, we do check for a file existing to allow us to further customize the branding and conda-forge probably do not need/want that. So CF could retake this lead!

Reply:
Actually, the conda-forge method does not incur extra run-time overhead, I just checked. The one thing it does do though is to patch the source code inside the build scripts and I do prefer patching is done via patch files as it's more obvious what's been changed that way.